helper_test.go

```diff
@@ -1161,6 +1165,74 @@ func createVariableSetVariable(t *testing.T, client *Client, vs *VariableSet, op
 	}
 }
 
+func waitForSVOutputs(t *testing.T, client *Client, svID string) {
+	t.Helper()
+	wg := &sync.WaitGroup{}
+	wg.Add(1)
+
+	go retry(func() (interface{}, error) {
+		outputs, err := client.StateVersions.ListOutputs(context.Background(), svID, nil)
+		if err != nil {
+			return nil, err
+		}
+
+		if len(outputs.Items) == 0 {
+			return nil, errors.New("no state version outputs found")
+		}
+
+		wg.Done()
+
+		return outputs, nil
+	})
+
+	wg.Wait()
+}
+
+func waitForRunLock(t *testing.T, client *Client, workspaceID string) {
+	t.Helper()
+	wg := &sync.WaitGroup{}
+	wg.Add(1)
+
+	go retry(func() (interface{}, error) {
+		ws, err := client.Workspaces.ReadByID(context.Background(), workspaceID)
+		if err != nil {
+			return nil, err
+		}
+
+		if !ws.Locked {
+			return nil, errors.New("workspace is not locked by run")
+		}
+
+		wg.Done()
+		return ws, nil
+	})
+
+	wg.Wait()
+}
+
+func retry(f retryableFn) (interface{}, error) {
+	tick := time.NewTicker(2 * time.Second)
```

Comment:
What's the number 2?

Reply:
It's effectively the quantity "multiplied" by the unit of time, so in this case it's 2 seconds.

Reply:
Cool! What do you think about moving the unit of time to a variable external to this loop, e.g., `TICKER_DURATION`?